libdnf/base/solver_problems.cpp

```diff
@@ -179,7 +179,7 @@ SolverProblems::SolverProblems(const SolverProblems & src) : p_impl(new Impl(*sr
 SolverProblems::~SolverProblems() = default;
 
 std::vector<std::vector<std::pair<libdnf::ProblemRules, std::vector<std::string>>>> SolverProblems::get_problems() {
-        return p_impl->package_solver_problems;
+        return p_impl->problems;
```

Comment:
Again, could you squash this commit with the original commit introducing the class and maintain a clean set of commits in the PR?

Reply:
I am really sorry, but it also creates merge problems. I would prefer to keep it like it is, or I can squash the PR into one commit. What do you think?